templates/ai/what-is-kubeflow.html

```diff
@@ -7,6 +7,9 @@
 {% block content %}
 
 <section class="p-strip u-no-padding--top u-no-padding--bottom">
+  {% if ("2022-08-16" | date_has_passed) and not ("2022-09-06" | date_has_passed) %}
+    {% include "shared/_charmed-kubeflow-beta-banner.html" %}
+  {% endif %}
   <div class="p-strip--suru-topped">
```

Comment:
The notification should be within the suru strip, it looks broken atm on /what-is-kubeflow:
```suggestion
  <div class="p-strip--suru-topped">
  {% if ("2022-08-16" | date_has_passed) and not ("2022-09-06" | date_has_passed) %}
    {% include "shared/_charmed-kubeflow-beta-banner.html" %}
  {% endif %}
```

Reply:
Fixed